### DISCUSSION TITLE
deepin-wine6-stable 6.0.0.54-1在Debian中存在兼容性问题

### DISCUSSION THREAD

**Discussion by YuruiHong** [linuxdeepin/developer-center#5043] [Troubleshooting | 故障排查]
目前deepin wine源 https://deepin-wine.i-m.dev 中的最新版deepin-wine6-stable在Debian运行微信等应用时，无法访问Linux文件管理器，导致上传文件等功能受到影响。针对微信这一个应用，我的解决方案是将最新版本3.7.0.30deepin24的deb中的依赖修改为大于等于6.0.0.46-1，并把deepin-wine6-stable锁定在该版本。未在包括deepin在内的其他发行版中尝试过是否有类似情况。如果问题比较普遍，希望开发者能对6.0.0.54-1进行修复，在此之前建议打包者包括最低6.0.0.46-1的依赖。

**Comment by BLumia** [comment]
你反馈错地方了，你所给出的链接不是 deepin 官方提供的软件源，是社区用户提供的仓库（ https://github.com/zq1997/deepin-wine ）。你应当反馈到那边。

**Reply by YuruiHong** [reply]
根据我的理解， https://github.com/zq1997/deepin-wine 维护的 https://deepin-wine.i-m.dev/ 应该就是deepin官方仓库的映射，用差量仓库的思想只留下必要的应用包，可访问 https://deepin-wine.i-m.dev/ 前端进行验证，如当前这里的 com.qq.weixin.deepin 指向的就是 https://com-store-packages.uniontech.com/appstore/pool/appstore/c/com.qq.weixin.deepin/com.qq.weixin.deepin_3.9.5.80deepin10_i386.deb

目前，6.0.0.58-1版本的deepin-wine仍然出现此问题，我依然通过修改依赖的方式使用46-1版本运行最新3.9.5.80的微信
